Seperate storage into assigments and permissions/roles (#70)
yiisoft/rbac: README.md

}
```

## Advanced usage:

> #### Create an instance and using other storage for user assignments

Reviewer: ```suggestion
### Using a separate storage for roles assignments
```

> if you want store user assignments in db or other storage

Reviewer: ```suggestion
Sometimes, it can be more convenient to separate roles assignments into a different storage than roles and permissions definitions, in order to allow more flexibility :

- Roles and permissions can be considered as "semi-static", as they only change when you update your application code, and are somehow tightly coupled to your code.

- Assignments, on the other hand, can be considered as "dynamic", and much more prone to changes, like when creating a new user, or updating its role, from within your app.

In order to handle assignments in your database, for example, you can use the following code:
```

Reviewer: Please fix lines widths when merging.

> For detail information - see [Issue 49](https://github.com/yiisoft/rbac/issues/49)

Reviewer: ```suggestion
```

@mikehaertl I borrowed your words for the guide, I hope you don't mind

```php
$manager = new Manager($storage, new ClassNameRuleFactory());

> $manager->setAssignmentStorage($otherStorage);

Reviewer: Please include $storage and $otherStorage instantiation code, and name the variables accordingly, to ease the comprehension for beginners ($phpStorage = .., $dbStorage = ..)

Author: Sorry, nut now we fon't have released implementation of rbac-db. After release - documentation will be updated.

}
```

## Advanced usage:

### Using a separate storage for roles assignments

> Sometimes, it can be more convenient to separate roles assignments into a different storage than roles and permissions definitions, in order to allow more flexibility :

Reviewer 2: ```suggestion
Sometimes, it makes sense to have role assignments in a different storage than roles and permission definitions:
```

> - Roles and permissions can be considered as "semi-static", as they only change when you update your application code, and are somehow tightly coupled to your code.
> 
> - Assignments, on the other hand, can be considered as "dynamic", and much more prone to changes, like when creating a new user, or updating its role, from within your app.

Reviewer 2: ```suggestion
- Roles and permissions could be considered "semi-static", as they only change when you update your application code.
- Assignments, on the other hand, could be considered "dynamic". They change more often: when creating a new user, or when updating user role from within your application.
```

> In order to handle assignments in your database, for example, you can use the following code:

Reviewer 2: ```suggestion
In order to store assignments in a database you can use the following code:
```
